### PR TITLE
Fix: JS API/Document/Repeater - Repeater items do not render after widget settings changed. ED-1651

### PR DESCRIPTION
**Pull request by iNewLegend**
Cannot test it, visual bug + tabs not rendered.

~Depends on https://github.com/elementor/elementor/pull/15034~

### CODE REVIEW

File: assets/dev/js/editor/container/container.js [hunk 1]
```diff
@@ -238,6 +238,15 @@ export default class Container extends ArgsObject {
...
 
+	/**
+	 * Function addRepeaterItem().
+	 *
+	 * @param {string} repeaterName
+	 * @param {Backbone.Model} rowSettingsModel
+	 * @param {number} index
+	 *
+	 * @returns {Container}
+	 */
```
Comment (StyleShit) [inline assets/dev/js/editor/container/container.js:249]: Nice JSDoc :)
Maybe add a function description, and maybe even parameters description if needed.

**Review by StyleShit — APPROVED** [review]
LGTM

**Review by StyleShit — APPROVED** [review]

File: assets/dev/js/editor/container/container.js [hunk 2]
```diff
@@ -238,28 +238,43 @@ export default class Container extends ArgsObject {
 		}
 	}
 
-	addRepeaterItem( repeaterName, rowSettingsModel, index ) {
-		let rowId = rowSettingsModel.get( '_id' );
+	/**
```
Comment (matipojo) [inline assets/dev/js/editor/container/container.js:241]: It looks like this file doesn't require any change to fix the issue,
Please send unrelated code via a separated PR.

**Review by matipojo — CHANGES_REQUESTED** [review]

File: assets/dev/js/editor/container/container.js (hunk 2 above)
Comment (iNewLegend) [reply assets/dev/js/editor/container/container.js:241]: The only 'logic' that changed in code is that it now does return the new container, are you sure we need separate PR for that?

Comment (matipojo) [reply assets/dev/js/editor/container/container.js:241]: Yes, a PR should contain only the changes that needed, in order to reflect exactly how the bug was solved.

**Review by matipojo — CHANGES_REQUESTED** [review]
Pleases pull from 3.3.0 & fix the PR name if needed.

File: assets/dev/js/editor/container/container.js (hunk 2 above)
Comment (iNewLegend) [reply assets/dev/js/editor/container/container.js:289]: Handled.

**Review by matipojo — CHANGES_REQUESTED** [review]
Please fix the conflct
